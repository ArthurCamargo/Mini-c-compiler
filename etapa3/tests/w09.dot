digraph G {
n0;
n0 -> n1;
n1;
n1 -> n2;
n2;
n2 -> n3;
n2 -> n4;
n2 -> n5;
n3;
n3 [label="a"];
n4;
n4 [label="1"];
n5;
n5 -> n6;
n5 -> n7;
n5 -> n8;
n6;
n6 [label="b"];
n7;
n7 [label="2"];
n8;
n8 -> n9;
n8 -> n10;
n8 -> n11;
n9;
n9 [label="c"];
n10;
n10 [label="3"];
n11;
n11 -> n12;
n12;
n12 -> n13;
n13;
n13 -> n14;
n14;
n14 [label="c"];
n13 [label="b"];
n12 [label="a"];
n11 [label="g"];
n8 [label="="];
n5 [label="="];
n2 [label="="];
n1 [label="f"];
n0 [label="g"];

}