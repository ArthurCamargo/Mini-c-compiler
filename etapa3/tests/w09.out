0x55a665685350
0x55a665685350, 0x55a665685510
0x55a665685510
0x55a665685510, 0x55a665685770
0x55a665685770
0x55a665685770, 0x55a6656856d0
0x55a665685770, 0x55a665685720
0x55a665685770, 0x55a665685880
0x55a6656856d0
0x55a6656856d0 [label="a"];
0x55a665685720
0x55a665685720 [label="1"];
0x55a665685880
0x55a665685880, 0x55a6656857e0
0x55a665685880, 0x55a665685830
0x55a665685880, 0x55a665685990
0x55a6656857e0
0x55a6656857e0 [label="b"];
0x55a665685830
0x55a665685830 [label="2"];
0x55a665685990
0x55a665685990, 0x55a6656858f0
0x55a665685990, 0x55a665685940
0x55a665685990, 0x55a665685a00
0x55a6656858f0
0x55a6656858f0 [label="c"];
0x55a665685940
0x55a665685940 [label="3"];
0x55a665685a00
0x55a665685a00, 0x55a665685a70
0x55a665685a70
0x55a665685a70, 0x55a665685ae0
0x55a665685ae0
0x55a665685ae0, 0x55a665685b50
0x55a665685b50
0x55a665685b50 [label="c"];
0x55a665685ae0 [label="b"];
0x55a665685a70 [label="a"];
0x55a665685a00 [label="g"];
0x55a665685990 [label="="];
0x55a665685880 [label="="];
0x55a665685770 [label="="];
0x55a665685510 [label="f"];
0x55a665685350 [label="g"];
